cloud/scope/cluster_test.go

```diff
@@ -713,3 +713,204 @@ func TestDeleteSubnet(t *testing.T) {
 		})
 	})
 }
+
+func TestCreateLoadBalancer(t *testing.T) {
+	var (
+		mockvpc  *mock.MockVpc
+		mockCtrl *gomock.Controller
+	)
+
+	setup := func(t *testing.T) {
+		t.Helper()
+		mockCtrl = gomock.NewController(t)
+		mockvpc = mock.NewMockVpc(mockCtrl)
+	}
+	teardown := func() {
+		mockCtrl.Finish()
+	}
```

Comment:
lets avoid sharing the content between the each Runs

```suggestion
	setup := func(t *testing.T) (*gomock.Controller, *mock.MockVpc) {
		t.Helper()
		return gomock.NewController(t), mock.NewMockVpc(mockCtrl)
	}
```

and you can call this setup as below:

```golang
mockController, mockVPC := setup(t)
t.Cleanup(mockController.Finish)
```

Reply:
Fixed.